### DISCUSSION TITLE
I found a bug

### DISCUSSION THREAD

**Discussion by guanyuyi666** [langgenius/dify#10580] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

I found a bug.  
In an ARM64 environment, starting the Dify Docker container results in the following error, and the program exits:

`[FATAL:partition_root.cc(863)] Check failed: (internal::SystemPageSize() == (size_t{1} << 12)) || (internal::SystemPageSize() == (size_t{1} << 14)).`

Solution: Upgrade `pypdfium2` to version "~4.30.0".

**Comment by AlphaHinex** [comment]
Download [pypdfium2-4.30.0-py3-none-manylinux_2_17_aarch64.manylinux2014_aarch64.whl](https://files.pythonhosted.org/packages/11/63/28a73ca17c24b41a205d658e177d68e198d7dde65a8c99c821d231b6ee3d/pypdfium2-4.30.0-py3-none-manylinux_2_17_aarch64.manylinux2014_aarch64.whl) from https://pypi.org/project/pypdfium2/#files , then use the Dockerfile below to build a new image:

```Dockerfile
FROM langgenius/dify-api:0.11.1
COPY pypdfium2-4.30.0-py3-none-manylinux_2_17_aarch64.manylinux2014_aarch64.whl /tmp
RUN pip install /tmp/pypdfium2-4.30.0-py3-none-manylinux_2_17_aarch64.manylinux2014_aarch64.whl
```

```bash
docker build -t langgenius/dify-api:0.11.1-arm .
```

**Reply by AlphaHinex** [reply]
https://github.com/langgenius/dify/pull/10942
